### DISCUSSION TITLE
Schema Version Mismatch Error when Using `cargo leptos watch` in WSL Ubuntu Environment

### DISCUSSION THREAD

**Discussion by MachoMaheen** [wasm-bindgen/wasm-bindgen#3908] [Q&A]
### Describe the Bug

When using `cargo leptos watch` command on a Rust project targeting wasm32-unknown-unknown in a WSL Ubuntu environment, a schema version mismatch error occurs between cargo-leptos and wasm-bindgen.

### Steps to Reproduce

1. Execute `cargo leptos watch` command on a Rust project targeting wasm32-unknown-unknown.
2. Observe the error message indicating a schema version mismatch between cargo-leptos and wasm-bindgen.

### Expected Behavior

I expected `cargo leptos watch` to compile the project successfully without any schema version mismatch errors.

### Actual Behavior

The `cargo leptos watch` command fails with the following error message:

Error: at /home/macho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-leptos-0.2.16/src/compile/front.rs:47:30
Caused by:
0: at /home/macho/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-leptos-0.2.16/src/compile/front.rs:122:10
1:

Additionally, the error message suggests trying to update the wasm-bindgen dependency or the wasm-bindgen-cli binary, both of which have been attempted without success.

### Additional Context

- Host environment: WSL Ubuntu
- Rust project setup: Using `cargo leptos watch` command on a project targeting wasm32-unknown-unknown.
- Attempts made to resolve the issue: Updating wasm-bindgen dependency and downgrading wasm-bindgen-cli binary according to the suggestions provided by the error message.
- No other errors or issues encountered during project compilation or execution.

This issue is hindering further development progress, and any assistance or guidance in resolving it would be greatly appreciated.
![Screenshot 2024-04-01 013753](https://github.com/rustwasm/wasm-bindgen/assets/99596115/1d7ce8fd-d73b-4ec8-9f18-dfb6d5cf2bae)

**Comment by daxpedda** [comment]
I'm assuming that updating the `wasm-bindgen` dependency somehow failed and not that the error message is wrong.

While compiling you should see which version is being compiled, you can also look into your `Cargo.lock` file; basically you should manually confirm the `wasm-bindgen` version used here.
